src/test/java/uk/ac/ebi/ena/webin/cli/context/genome/GenomeManifestReaderFileCountTest.java

```diff
@@ -18,36 +20,32 @@ public class GenomeManifestReaderFileCountTest {
 
   @Test
   public void testDefaultFileCount() {
+    /* We are keeping webin-cli-validator and sequencetools unchanged,
+    and hence we get AGP file type from GenomeManifest.FileType,
+    we exclude it upfront */
+    final List<GenomeManifest.FileType> fileTypeList =
+        Arrays.stream(GenomeManifest.FileType.values())
+            .filter(fileType -> !fileType.name().equals("AGP"))
+            .toList();
```

Comment:
better would be to create a private method and place this code inside it instead of repeating it along with the comments in multiple places.